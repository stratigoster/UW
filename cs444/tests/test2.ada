PACKAGE foo IS
  i : INTEGER := 3+4*5.0-3/2/1;
--                   ^^^
-- type mismatch: int * float
END foo;
