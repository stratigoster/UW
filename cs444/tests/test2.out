***ERROR***:2:21:	MUL  works on numeric values of the same type

------
Compilation error
------
